### DISCUSSION TITLE
How to get YOLO badge?

### DISCUSSION THREAD

**Discussion by GoodMan0127** [Schweinepriester/github-profile-achievements#357] [Q&A] [answered]
I can't find any reviewers when merging pull request.
What should I do?

**Comment by WHAHA-HA** [comment, marked as answer]
@GoodMan0127,
Under your repository name, click Pull requests. In the list of pull requests, click the pull request that you'd like to ask a specific person or a team to review. Navigate to Reviewers in the right sidebar. To request a review from a suggested person under Reviewers, next to their username, click Request.
You can see more info at [requesting-a-pull-request-review](https://docs.github.com/articles/requesting-a-pull-request-review)

**Reply by GoodMan0127** [reply]
thanks
